### PR TITLE
LPS-135269 test backend Creation

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
- LPS-135269 add test to check the behavior with portlets (widget pages)
- LPS-135269 add test to check the test "All languages hreflang meta tag will be displayed after disabling OTH from SEO settings"

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/layout/layout-seo-web-test/src/testIntegration/java/com/liferay/layout/seo/web/internal/servlet/taglib/test/OpenGraphTopHeadDynamicIncludeTest.java [hunk 1]
```diff
@@ -986,6 +986,43 @@ public void testIncludeLocalesLayoutTranslatedLanguagesOpenGraphDisabled()
...
+			mockHttpServletResponse.getContentAsString());
+
+		_assertMetaTag(document, "og:locale", _group.getDefaultLanguageId());
+		_assertAlternateLocalesTag(
+			document, _getAvailableLocalesLayoutTranslatedLanguages());
+
+		Stream<String> stream = Arrays.stream(
+			_layout.getAvailableLanguageIds());
+
+		Stream<Locale> localesStream = stream.map(LocaleUtil::fromLanguageId);
```
Comment (adolfopa) [inline modules/apps/layout/layout-seo-web-test/src/testIntegration/java/com/liferay/layout/seo/web/internal/servlet/taglib/test/OpenGraphTopHeadDynamicIncludeTest.java]: Chain the "collect" call here (e.g. stream.map(...).collect(...))

**Review by adolfopa — APPROVED** [review]

**Review by adolfopa — CHANGES_REQUESTED** [review]
There's a minor SF issue. See my previous comment.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
apps:layout:layout-seo-web-test:compileTestIntegrationJava is an unique failure from the [history](https://testray.liferay.com/zh/home/-/testray/cases/1330439891), re-run it to make sure it is a fluke.

**Comment by Hanlf** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by Hanlf** [comment]
[modules-compile-jdk8/0](https://testray.liferay.com/zh/home/-/testray/case_results/1557921946): flaky.
[modules-integration-mysql57-jdk8/0](https://testray.liferay.com/zh/home/-/testray/case_results/1557922027): [LPS-138527](https://issues.liferay.com/browse/LPS-138527)
[com.liferay.layout.set.prototype.exportimport.test.LayoutSetPrototypeExportImportTest.classMethod](https://testray.liferay.com/zh/home/-/testray/case_results/1557922027): [LPS-134190](https://issues.liferay.com/browse/LPS-134190)
The Poshi tests failures are known issues: [LRQA-69743](https://issues.liferay.com/browse/LRQA-69743), [LPS-135165](https://issues.liferay.com/browse/LPS-135165) and [LPS-138270](https://issues.liferay.com/browse/LPS-138270).

**Comment by Hanlf** [comment]
ci:forward

**Comment by austinchiang** [comment]
ci:reevaluate:1297401_133

**Comment by Hanlf** [comment]
The above failure is not related to this PR.
semantic-versioning-jdk8/0 failure is a known issue: [LPS-138527](https://issues.liferay.com/browse/LPS-138527)
In order to save CI resources, I've sent this PR to BChan directly: https://github.com/brianchandotcom/liferay-portal/pull/106549

cc/ @austinchiang

**Pull request closed without merging**